pkg/cmd/migrate.go

```diff
@@ -67,6 +69,19 @@ func migrateRun(cmd *cobra.Command, args []string) error {
 			log.Fatal().Err(err).Msg("unable to create migration driver")
 		}
 		manager = spannermigrations.SpannerMigrations
+	} else if datastoreEngine == "mysql" {
+		log.Info().Msg("migrating mysql datastore")
+
+		var err error
+		tablePrefix, err := cmd.Flags().GetString("datastore-mysql-table-prefix")
+		if err != nil {
+			log.Fatal().Msg(fmt.Sprintf("unable to get table prefix: %s", err))
+		}
+
+		migrationDriver, err = mysqlmigrations.NewMySQLDriverFromDSN(dbURL, tablePrefix)
+		if err != nil {
+			log.Fatal().Err(err).Msg("unable to create migration driver")
+		}
```

Comment:
@sbryant I believe there is a bug here: `manager` is not being set to the MySQL migration manager.

Reply:
follow up in https://github.com/authzed/spicedb/pull/550